usr/share/rear/backup/BORG/default/800_prune_old_backups.sh

```diff
@@ -3,16 +3,50 @@
 #
 # 800_prune_old_backups.sh
 
-if [ ! -z $BORGBACKUP_OPT_PRUNE ]; then
-    # Purge old archives according user settings.
-    LogPrint "Purging old Borg archives in repository $BORGBACKUP_REPO"
-    borg prune --list ${BORGBACKUP_OPT_PRUNE[@]} \
-    $BORGBACKUP_OPT_REMOTE_PATH $BORGBACKUP_OPT_UMASK \
-    --prefix ${BORGBACKUP_ARCHIVE_PREFIX}_ \
-    ${borg_dst_dev}${BORGBACKUP_REPO}
+# User might specify some additional output options in Borg.
+# Output shown by Borg is not controlled by `rear --verbose` nor `rear --debug`
+# only, if BORGBACKUP_SHOW_PROGRESS is true.
 
-    StopIfError "Failed to purge old backups"
+# shellcheck disable=SC2168
+local borg_additional_options=()
+
+BORGBACKUP_PRUNE_SHOW_PROGRESS=${BORGBACKUP_PRUNE_SHOW_PROGRESS:-$BORGBACKUP_SHOW_PROGRESS}
+BORGBACKUP_PRUNE_SHOW_STATS=${BORGBACKUP_PRUNE_SHOW_STATS:-$BORGBACKUP_SHOW_STATS}
+BORGBACKUP_PRUNE_SHOW_LIST=${BORGBACKUP_PRUNE_SHOW_LIST:-$BORGBACKUP_SHOW_LIST}
+BORGBACKUP_PRUNE_SHOW_RC=${BORGBACKUP_PRUNE_SHOW_RC:-$BORGBACKUP_SHOW_RC}
+
+is_true "$BORGBACKUP_SHOW_PROGRESS" && borg_additional_options+=( --progress )
+is_true "$BORGBACKUP_SHOW_STATS" && borg_additional_options+=( --stats )
+is_true "$BORGBACKUP_SHOW_LIST" && borg_additional_options+=( --list )
+is_true "$BORGBACKUP_SHOW_RC" && borg_additional_options+=( --show-rc )
+
+# https://github.com/rear/rear/pull/2382#issuecomment-621707505
+# Depending on BORGBACKUP_SHOW_PROGRESS and VERBOSE variables
+# 3 cases are there for `borg_prune` to log to rear log file or not.
+#
+# 1. BORGBACKUP_SHOW_PROGRESS true:
+#    No logging to rear log file because of control characters.
+#
+# 2. VERBOSE true:
+#    stderr (2) is copied to real stderr (8):
+#    2 is going to rear logfile
```

Comment:
The comment does no longer match what the code does because
now both stdout and stderr are copied to the ReaR log file
and redirected to the user's terminal.

Reply:
See a5948d5578bd5ab9155a4650259b8eed34e79c74.